### ISSUE TITLE
Virtual Interface

### ISSUE THREAD

**Issue by Medullitus** [morrownr/7612u#8]
Is virtual interface supported? Can fluxion and evil twins attack used without any problem?

**Comment by morrownr** [comment]
Hi @Medullitus 

Somehow I missed this. Sorry.

> Is virtual interface supported?

Yes

> Can fluxion and evil twins attack used without any problem?

I have not tested this.

Regards

**Comment by lab37** [comment]
NO, when fluxion  using 7612,  The program will get stuck and no task error prompt is given。

**Comment by morrownr** [comment]
What adapter/chipset does fluxon say works?
